### PR TITLE
New board

### PR DESCRIPTION
**Pull request by harbecke**
The board has colored edges now. These are identical to the GUI edges, so there is no overlapping and transposing the edges gives the edges of the opponent.

This only changes `Board.board_tensor` and not `Board.logical_board_tensor` in `logic.hexboard`. The models have a padding reduced by 1 in the first layer, so they are reusable, obviously they are not strong anymore.

Comparison to the old models is not possible automated. However, I finetuned the weights of the old model, and after only 1 day `11_2w4_2000` beat the website model 2-1 as starting player.

Screenshot_2020-04-30 TensorBoard This is the amazing puzzle loss function of the first half hour of finetuning. Furthermore, the model now employs the "walking the edge" strategy against itself, this might indicate that it is not as vulnerable against this strategy anymore. @simonant an evaluation against mohex would be much appreciated! @cleeff @simonant I hope this isn't a porting nightmare ;)

### CODE REVIEW

**Comment by cleeff** [comment]
Wow this is great! The puzzle loss curve is great. I'm glad you didn't give up half way through. I'll adapt the web app logic to include the boundary.
